### DISCUSSION TITLE
Support `.ansible.yml` extension for ansible files

### DISCUSSION THREAD

**Discussion by Tinyblargon** [renovatebot/renovate#26385] [Suggest an Idea]
### Tell us more.

The [official vscode extension for ansible](https://marketplace.visualstudio.com/items?itemName=redhat.ansible) uses the double `.ansible.yml` and `.ansible.yaml` extension. Currently Renovate has no way to detect files ending in those extensions. (I've found out where this should be implemented and can start on it immediately)

**Comment by secustor** [comment]
Feel free to open a PR if you already know exactly what to do.

**Comment by Tinyblargon** [comment]
Closed by #26390
